src/patternfly/utilities/Sizing/sizing.scss

```diff
@@ -41,49 +41,49 @@ $pf-u-height-options: (
   h-100vh:   (height 100vh)
 );
 
-@include pf-utility-builder($pf-u-width-options, $pf-global--breakpoint-list);
-@include pf-utility-builder($pf-u-height-options, $pf-global--breakpoint-list);
+@include pf-v5-utility-builder($pf-v5-u-width-options, $pf-v5-global--breakpoint-list);
+@include pf-v5-utility-builder($pf-v5-u-height-options, $pf-v5-global--breakpoint-list);
 
-.pf-u-min-width {
-  --pf-u-min-width--MinWidth: 0;
+.#{$pf-global}-u-min-width {
```

Comment:
Looks like  most/all utility classnames are broken - I think you meant to use `$pf-prefix` here